tools/compiletest/src/runtest.rs

```diff
@@ -395,7 +395,12 @@ impl<'test> TestCx<'test> {
     /// the expected output in `expected` file.
     fn run_expected_test(&self) {
         let proc_res = self.run_kani();
-        let expected_path = self.testpaths.file.parent().unwrap().join("expected");
+        let dot_expected_path = self.testpaths.file.with_extension("expected");
+        let expected_path = if dot_expected_path.exists() {
+            dot_expected_path
+        } else {
+            self.testpaths.file.parent().unwrap().join("expected")
+        };
```

Comment:
I'm fine with allowing multiple `expected` tests in the same folder, but not sure about why'd want to keep backwards compatibility. In my opinion, having two options may make it more confusing.

Reply:
I agree that only having the first would be simpler, but do we really want to rename all our existing expected files right now? Also, the `cargo-kani` mode does support both options.

Reply:
Yeah that renaming is why I didn't think it'd be a good idea completely change it. And they don't really interfere too much I think.

Reply:
Fair enough, we can rename later.